spec/decorators/case/sar/offender_complaint_decorator_spec.rb

```diff
@@ -92,13 +93,19 @@
   end
 
   describe "#highlight_flag" do
-    it "returns blank string when priority is normal" do
-      expect(offender_sar_complaint.highlight_flag).to eq ""
+    context "when Offender SAR Complaint - high priority" do
```

Comment:
This is happening in the offender complaint decorator, so it's not necessary to specify that this is an offender complaint.